### ISSUE TITLE
installing in windows causing error

### ISSUE THREAD

**Issue by knvpk** [benhoyt/graphyte#3]
Logs when installing the graphyte 1.2

```
Collecting graphyte
  Using cached graphyte-1.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\NECXFA~1\AppData\Local\Temp\pip-build-41uqbbiy\graphyte\setup.py", line 21, in <module>
        long_description = f.read()
      File "c:\program files\python35\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 2463: character maps to <undefined>

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\NECXFA~1\AppData\Local\Temp\pip-build-41uqbbiy\graphyte\
```

May be this is because the "" in the README.rst file

**Comment by knvpk** [comment]
For now I downloaded the package manually and cleared all the `"` in the README.rst file and installed manually

**Comment by benhoyt** [comment]
Hi @pavankumarkatakam -- thanks for the report. I believe I fixed this with commit 8e5b4442c26726105971c1c06154dfa06995efdc (and the new version 1.3 that I just uploaded to PyPI).
